LaCE-onboarding-guide.md

```diff
@@ -12,34 +12,85 @@ This page will help you solve common queries that you may face while working on
+
   - Canonical stories: In this section, the stories belonging to the topic are listed.
+  
+  ![Screenshot of canonical stories section](images/LaCEOnboardingGuide/topic-canonical-stories-section.png)
+  
 
 ## 2. Creating a Skill
   > **_NOTE:_** For a skill to be visible on the practice tab of a topic, it must have at least 10 questions associated with it.
 - A skill can be directly created from the topic editor by clicking the `ADD SKILL` button under `Subtopics` section of the topic editor. But in this case, the skill will be automatically assigned to that topic.
 - A skill can also be created from the Topic and Skill Dashboard as well but it will not be assigned to any topic.
 - On clicking the `ADD SKILL` option a modal will pop up with necessary details of the skill.
+
+![Screenshot of creating a skill](images/LaCEOnboardingGuide/creating-a-skill.png)
+
 - Once this is done, the skill is created and you will be redirected to the skill editor page
 - The main skill editor section has the following part:
   - Details
+  
+  ![Screenshot of skill details section](images/LaCEOnboardingGuide/skill-details-section.png)
+
   - Worked example
-  - Misconceptions: Common errors that users may have and their explanations.
-  - Pre-requisite Skills: Skills that are to be completed before attempting the current skill.
+
+  ![Screenshot of worked examples section](images/LaCEOnboardingGuide/skill-worked-examples-section.png)
+
+  - Misconceptions: Common errors that users may have and their explanations. The creator is given the option to enforce the misconception on all question within the skill as seen in the image below.
+ 
+  ![Screenshot of skill misconceptions section](images/LaCEOnboardingGuide/skill-misconceptions-section.png)
+
+  - Pre-requisite Skills: Skills that are to be completed before attempting the current skill. The prerequisite skills can be filtered by topic, subtopic and by title of the skill.
+
+  ![Screenshot of pre-requisite skill section](images/LaCEOnboardingGuide/skill-pre-requisite-skill-section.png)
+
+  - Rubrics: This section of the skill helps question creators make questions under a certain rubric. 
+  
+  ![Screenshot of skill rubrics section](images/LaCEOnboardingGuide/skill-rubrics-section.png)
+
 - The questions section lists the questions in the skill:
   - To add a new question, select the `ADD QUESTION` option
   - In the question editor, there are 5 sections:
     - Difficulty
+    
+    ![Screenshot of question difficulty section](images/LaCEOnboardingGuide/question-difficulty.png)
+
     - Problem
+    
+    ![Screenshot of problem section](images/LaCEOnboardingGuide/question-problem.png)
+
     - Interaction
-    - Answers ans Responses
+    
+    ![Screenshot of interaction and answers and responses sections](images/LaCEOnboardingGuide/question-interaction-answers-responses.png)
+
+    - Answers ans Responses (This section can be seen in the image above)
```

Comment:
ans --> and

Reply:
Hi @seanlip , I have made the change. Thanks!